.github/workflows/doc-tests.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       uses: actions/setup-dotnet@v4
       with:
         dotnet-version: 6.0.x
```

Comment:
Probably want to delete the first copy :)

Reply:
I've tried various combinations and the one that works is to install both 6 and 8

Reply:
Oh! So it's something about building and running with different versions?

In that case add comments instead. As is it just looks unintentional and someone will 100% remove it later.